tests/httpInterface.patchBulk.test.js

```diff
@@ -574,6 +574,64 @@ tap.test('HTTP PATCH /bulk', t => {
       t.end()
     })
 
+    t.test('$addToSet', async t => {
+      const DOC_TEST = {
+        ...fixtures[0],
+        metadata: {
+          somethingString: 'the-saved-string',
+          somethingArrayOfNumbers: [VALUE_AS_NUMBER],
+        },
+        attachments: [{
+          name: 'note',
+          neastedArr: [123, VALUE_AS_NUMBER],
+        }],
+      }
+      const UPDATE_COMMAND = {
+        $addToSet: {
+          'attachments.0.neastedArr': VALUE_AS_STRING,
```

Comment:
Just to understand: from the test I'm observing the `$addToSet` operator considers `42` and `'42'` as being the same value. Is this due to Javascript or is a MongoDB behavior?